### PR TITLE
Add deprecation schedule

### PR DESCRIPTION
**Pull request by sandersn**
and explain tags a bit more fully

### CODE REVIEW

**Comment by galkin** [comment]
Awesome!

**Comment by sandersn** [comment]
@galkin I simplified "2 years" to actually be "25 months" so that the table looked more uniform. In practise I think the deprecation will be unnoticed by almost everyone so the precise day of the month won't matter.

File: README.md [hunk 1]
```diff
@@ -64,25 +64,43 @@ If you still can't find it, check if it [bundles](http://www.typescriptlang.org/
...
-Packages that existed before November 2019 may have older versions that are explicitly marked compatible with older versions of Typescript; use the tag "ts2.6" for Typescript 2.6, for example.
-
-For example, if you run `npm dist-tags @types/react`, you'll see the following table that shows that react@16.4 has types for Typescript 2.6:
+Here is the support window:
+
+Version | Released | End of Support
+-- | -- | --
+2.8 | March 2018 | March 2020
+2.9 | May 2018 | May 2020
+3 | July 2018 | July 2020
```
Comment (minestarks) [inline README.md]: 3.0?

**Comment by minestarks** [comment]
LGTM

**Review by minestarks — APPROVED** [review]

**Pull request merged**